packages/web-app/app/_providers/wagmi-config.ts

```diff
@@ -12,6 +12,7 @@ export const wagmiConfig = getDefaultConfig({
   appName: 'Citizend',
   projectId: process.env.NEXT_PUBLIC_WC_PROJECT_ID,
   chains: [
+    sepolia,
```

Comment:
It is already below 🤔

Reply:
I actually used `switchChain` to force the app to connect to sepolia, instead of defaulting to the chain in first position of this list, so this change is not needed